forge-gui/res/cardsfolder/upcoming/rise_of_the_varmints.txt

```diff
@@ -0,0 +1,8 @@
+Name:Rise of the Varmints
+ManaCost:3 G
+Types:Sorcery
+A:SP$ Token | TokenAmount$ Count$ValidGraveyard Creature.YouCtrl | TokenScript$ g_2_1_varmint | SpellDescription$ Create X 2/1 green Varmint creature tokens, where X is the number of creature cards in your graveyard.
```

Comment:
@tool4ever what do you think? should we use SVars for the Amount or not?

Reply:
I think both in general ways are ok/safe?
sometimes one or the other might be easier to read